test/supervisor_test.exs

```diff
@@ -89,6 +89,28 @@ defmodule SupervisorTest do
       {:ok, members} = Horde.Cluster.members(:init_sup_test_1)
       assert 2 = Enum.count(members)
     end
+
+    test "can use `child_spec` function to override defaults from Horde.Supervisor" do
+      spec = %{
+        id: 123,
+        start:
+          {TestSupervisor3, :start_link,
+           [
+             custom_id: 123,
+             name: :init_sup_test_3,
+             strategy: :one_for_one
+           ]},
+        restart: :transient,
+        type: :supervisor
+      }
+
+      assert spec =
```

Comment:
This assertion isn't asserting anything I don't think. Should be `^spec` I think.

Reply:
Ohhh, it was testing... nothing  🙈 Already fixed!